html_templates/footer.html

```diff
@@ -0,0 +1,28 @@
+
+		</div>
+	</div>
+</div>
+</div>
+
+<div id="footer">
+  <div class="container">
+  	<div class="row">
+	<div class="col-md-6">
+    <p class="text-muted">&copy; Copyright 2015 Novoda Ltd., All Rights Reserved.</p>
```

Comment:
What I mean is, will you remember changing the footer in 8 months with the new year? Why not having this filled in with the year at time of generating the diff?